packages/drawer/src/views/DrawerView.tsx

```diff
@@ -88,8 +88,12 @@ export default function DrawerView({
   sceneContainerStyle,
 }: Props) {
   const [loaded, setLoaded] = React.useState([state.index]);
+
+  const { width, height } = Dimensions.get('window');
+  const isBigScreen = Math.min(width, height) >= 768;
```

Comment:
I think maybe we can leave the automatic switching for now since now it's stable and this can be considered a breaking change. Let the user pass the `drawerType` explicitly